src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -1877,20 +1881,20 @@ namespace OpenMS
         if (!ms_run_in_data.empty())
         {
           // prepend file:// if not there yet
-          for (const String& s : ms_run_in_data)
+          for (const String &s : ms_run_in_data)
```

Comment:
do we prefer String &s over String& s ?

Reply:
ah good catch. have to change my editor settings